test/client-errors.js

```diff
@@ -268,7 +268,7 @@ test('POST which fails should error response', (t) => {
   t.plan(4)
 
   const server = createServer()
-  server.once('request', (req, res) => {
+  server.on('request', (req, res) => {
```

Comment:
This test shouldn't have passed before... some other bug that got resolved with this PR.